src/components/Footer.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+const Footer = () => {
+  return (
+    <footer>
+      <ul>
+        <li>
+          <a href="/en-savoir-plus">En savoir plus</a>
+        </li>
+        <li>
+          <a href="/contact">Nous contacter</a>
+        </li>
```

Comment:
Replace links with "link to"

Reply:
Ok, done for the next push (with title attributes inside, that I forgot before).

Reply:
You can see that modification on this commit: https://github.com/WildCodeSchool/remotefr-js-0920-p2-themoviedb/pull/1/commits/66840d2c1332d317c1b41cfd0c77f239022663a2